src/v0/destinations/zendesk/transform.js

```diff
@@ -405,6 +406,9 @@ async function getUserMembershipPayload(message, headers, orgId, destinationConf
 }
 
 async function createOrganization(message, category, headers, destinationConfig, baseEndpoint) {
+  if (!isDefinedAndNotNull(message.traits)) {
```

Comment:
is this mandatory?

Reply:
yes it is. Because, on the basis of traits mapping we are creating the organisation later on. So if trait object is not there, it will be of no use. Anyway the events not having traits were failing, so no chance of data loss